pom.xml

```diff
@@ -48,28 +49,39 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.235.x</artifactId>
-                <version>11</version>
+                <artifactId>bom-2.249.x</artifactId>
+                <version>950.v396cb834de1e</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
             <dependency>
-                <groupId>com.fasterxml.jackson.core</groupId>
-                <artifactId>jackson-databind</artifactId>
-                <version>2.11.1</version>
+                <!-- fix upper bound -->
```

Comment:
What is this for? Something we can address in `bom`?

Reply:
enforcer error without it is
```
Require upper bound dependencies error for joda-time:joda-time:2.8.1 paths to dependency are:
+-io.jenkins.plugins:aws-global-configuration:1.7-SNAPSHOT
  +-org.jenkins-ci.plugins.aws-java-sdk:aws-java-sdk-minimal:1.12.70
    +-com.amazonaws:aws-java-sdk-core:1.12.70
      +-joda-time:joda-time:2.8.1
and
+-io.jenkins.plugins:aws-global-configuration:1.7-SNAPSHOT
  +-io.jenkins.configuration-as-code:test-harness:1.52
    +-com.github.erosb:everit-json-schema:1.13.0
      +-com.damnhandy:handy-uri-templates:2.1.8
        +-joda-time:joda-time:2.10.2
```

Reply:
Hmm, so I think we need to add at least `aws-java-sdk-minimal` to the `bom`.